python/GafferUI/NumericWidget.py

```diff
@@ -141,6 +142,19 @@ def __keyPress( self, widget, event ) :
 
 		return False
 
+	def __wheel( self, widget, event ) :
+
+		assert( widget is self )
+
+		if not self.getEditable() :
+			return False
+
+		scaleFactor = 15.0
+
+		self.__incrementIndex( self.getCursorPosition(), event.wheelRotation / scaleFactor )
```

Comment:
I have a suggestion which fixes this but also removes any concerns about different mouse-wheel speeds :

`self.__incrementIndex( self.getCursorPosition(), int( math.copysign( 1, event.wheelRotation ) ) )`

Reply:
It might just be my setup but the downside to "any scroll amount is considered one" is that I see 3-5 wheel events for any one "step" of my scroll wheel so I'm unable to make fine adjustments. I'm happy to blame it on my mouse if this is working acceptably for you both...